packages/mdc-button/_mixins.scss

```diff
@@ -386,9 +387,12 @@ $query: mdc-feature-all()) {
 
   @include mdc-typography(button, $query);
   @include mdc-button-horizontal-padding($mdc-button-horizontal-padding, $query);
+  @include mdc-elevation-overlay-parent($query: $query);
```

Comment:
Can we rename this to `mdc-elevation-overlay-position()`? seems more appropriate.

WDYT?

Reply:
I named it `parent` because it should be applied to the parent element, not the overlay itself. What about `mdc-elevation-overlay-parent-position`?

Reply:
Gotcha. `mdc-elevation-overlay-parent-position` makes more sense in that case.